TeraProxyGUI.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+./node_modules/electron/dist/electron --high-dpi-support=1 --force-device-scale-factor=1 --js-flags="--use-strict" ./bin/index.js &
```

Comment:
Debatable whether this script is useful at all. Background processes die when the shell closes, afaik, so you can only run this from within a terminal anyway (where the shell persists and if your source the script instead of executing it) and then you might as well use the "with console" variant in the first place.

When you want to launch something with a GUI but without a terminal, you'd use a `.desktop` file instead.